newIDE/app/src/Profile/AuthenticatedUserContext.js

```diff
@@ -79,6 +79,21 @@ export const initialAuthenticatedUser = {
   getAuthorizationHeader: () => Promise.reject(new Error('Unimplemented')),
 };
 
+export const authenticatedUserLoggedOutAttributes = {
+  authenticated: false,
+  firebaseUser: null,
+  profile: null,
+  loginState: 'done', // Particularly useful on first mount.
```

Comment:
I have a feeling this comment will be hard to understand in a few weeks/months. Something like "A logged out user is not the same a user being loaded (which can be the case at startup). Use this loginState to make sure this is understood by the app as a user logged out, and not loading."